docs/cue/reference/components/sinks/aws_sqs.cue

```diff
@@ -89,7 +89,24 @@ components: sinks: aws_sqs: components._aws & {
 			type: string: {
 				default: null
 				examples: ["vector", "vector-%Y-%m-%d"]
-				syntax: "literal"
+				syntax: "template"
+			}
+		}
+		message_deduplication_id: {
+			common:      false
+			description: """
+			The message deduplication ID to allow AWS to identify duplicate messages. This field should be unique
+			between events.
```

Comment:
I think this could use some clarification.
```suggestion
			The message deduplication ID value to allow AWS to identify duplicate messages.
			This value is a template which should result in a unique string for each event.
```
(assuming I haven't misunderstood what this is meant to accomplish)